test/radix/quic_tests.c

```diff
@@ -24,10 +24,25 @@ DEF_SCRIPT(simple_conn, "simple connection to server")
     OP_READ_EXPECT_B(C, "orange");
 }
 
+DEF_SCRIPT(simple_thread_child,
+           "test that RADIX multithreading is working (child)")
+{
+}
+
+DEF_SCRIPT(simple_thread,
+           "test that RADIX multithreading is working")
+{
+    size_t i;
+
+    for (i = 0; i < 2; ++i)
+        OP_SPAWN_THREAD(simple_thread_child);
```

Comment:
do you need a command macro here to join on the threads you create?  Or is that manageable in a future update?

Reply:
This is done automatically by `quic_bindings.c` in `RADIX_PROCESS_join_all_threads`.

If we need to control joining explicitly in a test script, that can easily be added later. Just the framework for now.